test/errors/multipleQuestionArgs-detailed.good

```diff
@@ -0,0 +1,29 @@
+─── error in multipleQuestionArgs.chpl:4 [MultipleQuestionArgs] ───
+  Cannot have ? more than once in a call
```

Comment:
Can you even have one in a normal calling context like this?  To me, `?` is only intended for use in type constraint contexts, not for general procedure calls.  Is there a problem where we don't have enough context to be able to distinguish that at the point where this error is generated?  If `test()` just took one argument and was called with just one `?` what would/should happen?